client/signup/steps/pressable-store/index.jsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import EmailValidator from 'email-validator';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Gridicon from 'components/gridicon';
+
+import LoggedOutForm from 'components/logged-out-form';
+import LoggedOutFormFooter from 'components/logged-out-form/footer';
+import LoggedOutFormLinks from 'components/logged-out-form/links';
+import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
+import FormTextInput from 'components/forms/form-text-input';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import FormSectionHeading from 'components/forms/form-section-heading';
+
+import renderHeroImage from './hero-image';
+
+export default React.createClass( {
+	displayName: 'PressableStoreStep',
+
+	propTypes: {
+		stepName: React.PropTypes.string.isRequired,
+		goToNextStep: React.PropTypes.func.isRequired,
```

Comment:
This prop is required, yet never used, and the final `render` method even intentionally overrides it with `{ ...this.props } goToNextStep={ undefined }`. Is that transitory?